code-experiments/src/logger_bbob.c

```diff
@@ -426,12 +443,14 @@ static void logger_bbob_evaluate(coco_problem_t *problem, const double *x, doubl
        * does not correspond to evaluation counter 1.
        */
       logger_bbob_write_data(logger->fdata_file, 
-          1, initial_solution_fvalue, initial_solution_fvalue, 
-          logger->optimal_fvalue, inner_problem->initial_solution, problem->number_of_variables);
+          1, 0, initial_solution_fvalue, initial_solution_fvalue,
```

Comment:
I don't see why `0` is correct here. Shouldn't it be `logger->number_of_evaluations_constraints`?

Is there a problem with the first entry having non-zero or even a large constraints evaluation counter? I wouldn't be entirely comfortable with ignoring them all together. The post-processing should be able to deal with that, as the code to write in these numbers needs yet to be written anyways.

Reply:
It was
```
logger_bbob_write_data(logger->fdata_file, 
    1, initial_solution_fvalue, initial_solution_fvalue, 
    logger->optimal_fvalue, inner_problem->initial_solution,
    problem->number_of_variables);
```
before. And since the test in line ```432``` is
```
if (logger->number_of_evaluations == 1 && problem->number_of_constraints > 0)
```
I assumed there should be ```0``` there. I think there was an issue with the first entry being non-zero. Maybe @brockho remembers better.

Reply:
I remember there was an issue that the evaluation count in the first output line should/must be 1. As the constraints evaluation count column is newly introduced, there should be no constraint there to begin with, unless we constrain ourselves.

Reply:
See also https://github.com/numbbo/coco/issues/1518#issuecomment-327840025